she_codes_news/users/templates/users/customuser_detail.html

```diff
@@ -0,0 +1,12 @@
+{% extends base.html %}
+
+{% block content %}
+
+<h2>{{ customuser.username }}</h2>
+
+{% for story in CustomUser.NewsStory_set.all %}
```

Comment:
nice! 
normally expect this to be lowercase though, i guess it must work either way.